README.md

```diff
@@ -10,7 +10,7 @@ GitHub, anyone can quickly try your Fiddle out by just entering it in the
 address bar.
 
 <h3 align="center">
-  <a href="https://github.com/electron/fiddle/releases/tag/0.1">
+  <a href="https://github.com/electron/fiddle/releases/">
```

Comment:
There's a magic link we can use here:

`https://github.com/electron/fiddle/releases/latest`

Reply:
Ohh, nice!